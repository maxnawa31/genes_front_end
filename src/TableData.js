import React from 'react';
import {StyledTableData} from './StyledComponents'

const TableData = props => {
  return <StyledTableData>{props.data}</StyledTableData>;
};

export default TableData;
