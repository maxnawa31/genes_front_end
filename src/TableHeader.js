import React from 'react';
import { StyledHeader } from './StyledComponents';

const TableHeader = props => {
  return <StyledHeader>{props.header}</StyledHeader>;
};

export default TableHeader;
