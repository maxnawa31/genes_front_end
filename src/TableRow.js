import React from 'react';
import { StyledRow } from './StyledComponents';

const TableRow = props => {
  return <StyledRow>{props.children}</StyledRow>;
};

export default TableRow;
