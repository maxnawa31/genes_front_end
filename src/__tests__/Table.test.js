import React from 'react';
import Table from '../Table';
import TableRow from '../TableRow';
import TableData from '../TableData';
import TableHeader from '../TableHeader';
import renderer from 'react-test-renderer';

describe('<Table/>', () => {
  it('matches the snapshot', () => {
    const tree = renderer.create(<Table/>).toJSON();
    expect(tree).toMatchSnapshot();
  })
})

describe('<TableRow/>', () => {
  it('matches the snapshot', () => {
    const tree = renderer.create(<TableRow/>).toJSON();
    expect(tree).toMatchSnapshot();
  })
})

describe('<TableData/>', () => {
  it('matches the snapshot', () => {
    const tree = renderer.create(<TableData/>).toJSON();
    expect(tree).toMatchSnapshot();
  })
})

describe('<TableHeader/>', () => {
  it('matches the snapshot', () => {
    const tree = renderer.create(<TableData/>).toJSON();
    expect(tree).toMatchSnapshot();
  })
})