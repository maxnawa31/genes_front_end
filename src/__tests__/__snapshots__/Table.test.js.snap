// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Table/> matches the snapshot 1`] = `
<table
  className="sc-EHOje eKPvmx"
>
  <tbody />
</table>
`;

exports[`<TableData/> matches the snapshot 1`] = `
<th
  className="sc-gzVnrw ibZhkv"
/>
`;

exports[`<TableHeader/> matches the snapshot 1`] = `
<th
  className="sc-gzVnrw ibZhkv"
/>
`;

exports[`<TableRow/> matches the snapshot 1`] = `
<tr
  className="sc-bZQynM hyBRIN"
/>
`;
